tests/tracking/_tracking_service/test_utils.py

```diff
@@ -155,6 +168,31 @@ def test_get_store_sqlalchemy_store_with_artifact_uri(tmp_wkdir, db_type):
     mock_create_engine.assert_called_once_with(uri, pool_pre_ping=True)
 
 
+def test_get_store_sqlalchemy_store_with_artifact_root():
```

Comment:
Would it be possible to convert the 3 tests (+ this one) that share almost the same code into a parameterized test?

Reply:
No reason indeed, totally agree

Reply:
Done